### DISCUSSION TITLE
Dexcom ONE: Calibration . « - . Sensor data fails sanity test - Cannot Calibrate! raw:-0.1279

### DISCUSSION THREAD

**Discussion by Agio56** [NightscoutFoundation/xDrip#2907] [Q&A]
I have attempted to calibrate my sensor which has been in for about 30 hours. Meter blood glucose was 84, xDrip reported 95.
In the log I found these entries that I can not understand.
What does it mean?

I've attached a screenshot of the log if it helps
![Screenshot_20230627-132608-1](https://github.com/NightscoutFoundation/xDrip/assets/137171485/0cd0247b-49cb-404e-84d1-1547c2c5a89b)

**Comment by Navid200** [comment]
Those two values, 84 and 95, are very close.  Why did you calibrate?
How do you know your meter is more accurate?
Please have a look at this, where the manufacturer explains why the meter and G6 may not be identical:  https://www.dexcom.com/faqs/is-my-dexcom-sensor-accurate

xDrip submits the calibration to the transmitter.  You can ignore the logs.

**Reply by Agio56** [reply]
"xDrip submits the calibration to the transmitter. You can ignore the logs."

Is this why "calibration data table" is completely empty? No values and no charts, ever

**Reply by Navid200** [reply]
xDrip shows calibration graph and table when it has access to raw values.  With a regular G6, xDrip has access to no raw values.

Even if xDrip has access to raw values, and shows a calibration graph and table, does not necessarily mean it is using those.  If you use Native mode, it is not.

**Reply by Agio56** [reply]
Perfect. Thank you. You are a valuable source of information!!

**Reply by Navid200** [reply]
You're welcome.  Thanks
